ff --git a/tests/create_tests_source.py b/tests/create_tests_source.py
index 1feb1443..2c5560f4 100755
--- a/tests/create_tests_source.py
+++ b/tests/create_tests_source.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python3
 
 """ Create a simple stereo file containing a sine tone at 441 Hz, using only
 python's built-in modules. """
diff --git a/tests/wscript_build b/tests/wscript_build
index c99a051e..d3fd0633 100644
--- a/tests/wscript_build
+++ b/tests/wscript_build
@@ -13,7 +13,7 @@ test_sound_abspath = bld.path.get_bld().make_node(test_sound_target)
 test_sound_abspath = str(test_sound_abspath).replace('\\', '\\\\')
 
 b = bld(name='create_tests_source',
-    rule='python ${SRC} ${TGT}',
+    rule='python3 ${SRC} ${TGT}',
     source='create_tests_source.py',
     target=test_sound_target)
 # use post() to create the task, keep a reference to it

